Makefile

```diff
@@ -36,5 +41,8 @@ thumbnails: ${board}/board.json
 
 ${moment}.ttl:${board}/board.json
 	./trello2moment --board=${board} 2>${moment}.err > ${moment}_t.ttl
-	${riot} --formatted=ttl --base=z: ${moment}_t.ttl  | sed -e 's/<z:/</g' > $@
+	riot --formatted=ttl --base=z: ${moment}_t.ttl  | sed -e 's/<z:/</g' > $@
```

Comment:
@ldragoon I hate to keep on this, but now you are not using the variable you described above.